packages/govuk-frontend/package.json

```diff
@@ -6,6 +6,7 @@
   "files": [
     "src",
     "dist",
+    "govuk-prototype-kit.config.json",
```

Comment:
This ensures GOV.UK Prototype Kit can find the config

But paths at their end are still hard coded: https://github.com/alphagov/govuk-prototype-kit/commit/953830afb1f2da8bdfac0a733f3a578a8fb1bbac